components/host-dashboard/AddFundsModal.js

```diff
@@ -303,7 +303,7 @@ const AddFundsModal = ({ host, collective, ...props }) => {
   // From the Host dashboard we pass host and collective as API v2 objects
   const canAddHostFee = host.plan?.hostFees && collective.id !== host.id;
   const defaultHostFeePercent = canAddHostFee && collective.hostFeePercent ? collective.hostFeePercent : 0;
-  const canAddPlatformTip = collective.host?.isTrustedHost;
+  const canAddPlatformTip = get(data, 'account.host.isTrustedHost');
```

Comment:
What's the benefit of using the `get` utility  instead of optional chaining like it was before?

Reply:
My thinking is that if we were to use optional chaining it should could be something like `data.account.host?.isTrustedHost`. I prefer to use `get` if it's more than three levels deep so as to avoid intermediary values being `null`. For example, `account` might be null in certain cases (if that's the case we have to use `data.account?.host?.isTrustedHost`); we avoid this by using `get`. Just a personal preference, but if you think this doesn't warrant the use of `get` let me know; I am okay with changing it to optional chaining.

Reply:
I've changed it to use optional chaining. 👍🏽